mock-hpos-api/defaultResponse.js

```diff
@@ -0,0 +1,56 @@
+const _ = require('lodash')
+
+// mock of normal hpos api responses
+
+const happs = [{
+  id: 1,
+  name: 'HoloFuel'
+}, {
+  id: 2,
+  name: 'Community'
+}, {
+  id: 3,
+  name: 'Elemental Chat'
+}]
+
+
+const data = {
+  get: {   
+    '/hosted_happs': {
+      hosted_happs: happs
+    },
+    '/config': {
+      admin: {
+        public_key: '5m5srup6m3b2iilrsqmxu6ydp8p8cr0rdbh4wamupk3s4sxqr5',
+        email: 'test@test.com'
+      },
+      holoportos: {
+        network: 'test',
+        sshAccess: true
+      },
+      name: "Lana Wilson's HP"
+    }
+  },
+  put: {
+    '/config': args => args
+  }
+}
+
```

Comment:
You would also want a separate one to toggle.

To enable: PUT `/profiles/development/features/ssh`
To disable: DELETE `/profiles/development/features/ssh`

Reply:
👍